config/samples/airlock_v1alpha1_mongodbcluster.yaml

```diff
@@ -38,6 +38,22 @@ spec:
   # Optional. If this is set, along with useAtlasApi, all the kubernetes nodes on the cluster will be added to the Atlas firewall. The only available value right now is "rancher-annotation", which uses the rke.cattle.io/external-ip annotation.
   atlasNodeIpAccessStrategy: rancher-annotation
 
+  atlasScheduledAutoscaling:
+    # Whether the autoscaling is enabled or not. Defaults to false.
+    enabled: true
+    
+    # The low tier the cluster will scale down to. Defaults to "M20".
+    lowTier: "M10"
+    
+    # The high tier the cluster will scale up to. Defaults to "M50".
+    highTier: "M20"
+
+    # The cron expression that will be used to scale down the cluster. Defaults to "0 20 * * 1-5".
+    scaleDownCronExpression: "* * * * *"
+
+    # The cron expression that will be used to scale up the cluster. Defaults to "0 6 * * 1-5".
+    scaleUpCronExpression: "0 0 * * *"
```

Comment:
I love the word use of `cronexpression` over just `expression` - small +ve comment.